dothething/dothething/Assets.xcassets/Placeholder.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "obama.jpg",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```

Comment:
The changes made to this file are related to adding a new image asset for a placeholder image. This is a common practice when working with image assets in an iOS app, and the changes look correct. The added JSON content defines the three different scales (1x, 2x, and 3x) of the "obama.jpg" image, which is the placeholder image for the app. This ensures that the image will be displayed correctly on devices with different screen resolutions.